pkg/ccl/backupccl/backup_planning.go

```diff
@@ -399,6 +353,168 @@ func checkPrivilegesForBackup(
 	return nil
 }
 
+// privilegesDeprecationNotice returns a notice that outlines the deprecation of
+// the existing privilege model for backups, and the steps to take to adopt the
+// new privilege model, based on the type of backup being run.
+func privilegesDeprecationNotice(
+	p sql.PlanHookState, backupStmt *annotatedBackupStatement, targetDescs []catalog.Descriptor,
+) string {
+	if backupStmt.Targets == nil {
+		return ""
+	}
+
+	// If a user is running `BACKUP DATABASE` buffer all the databases that will
+	// require the `BACKUP` privilege.
+	var notice string
+	preamble := "The existing privileges required for backup are being deprecated " +
+		"in favour of a fine-grained privilege model explained here <link>. In 22.2 we will support " +
+		"both the old and new privilege requirements, but in 23.1 to run"
+	if backupStmt.Targets.Databases != nil {
+		dbsRequireBackupPrivilege := make([]string, 0)
+		for _, desc := range targetDescs {
+			switch desc := desc.(type) {
+			case catalog.DatabaseDescriptor:
+				dbsRequireBackupPrivilege = append(dbsRequireBackupPrivilege, desc.GetName())
+			}
+		}
+
+		notice = fmt.Sprintf("%s `BACKUP DATABASE`, user `%s` will exclusively require the "+
+			"`BACKUP` privilege on database %s.",
```

Comment:
I know we do it in lots of places, but the terminal doesn't format backticks and if someone errantly copies and pasts them, it can create confusions since the terminal _will_ execute the contexts of backtics in a subshell.

Reply:
makes sense, removed backticks